src/SFA.DAS.LevyTransferMatching.Web/Models/Cache/CreatePledgeCacheItem.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Cache
+{
+    public class CreatePledgeCacheItem
+    {
+        public CreatePledgeCacheItem()
+        {
+            Key = Guid.NewGuid();
+        }
+
+        public CreatePledgeCacheItem(Guid key)
+        {
+            Key = key;
+        }
+
+        public Guid Key { get; set; }
```

Comment:
Would in-lining the initialisation of the property and removing the ctor make it more readable?
```suggestion
        public Guid Key { get; set; } = Guid.NewGuid();
```

Reply:
Will take a look - not sure; there are two ctors here